### ISSUE TITLE
data transform does not behave as expected

### ISSUE THREAD

**Issue by crlna16** [AI4EO/tum-planet-radearth-ai4food-challenge#4]
In data_transform.py, the functions np.flipud / np.fliplr are applied to 4D arrays of dimension [TIME, CHANNELS, WIDTH, HEIGHT]. I think that these functions flip the 4D array along the first two axes, not along the last 2D axes as expected.

In contrast, the np.rot90 function in the same module is correctly applied along the last two axes of the 4D array.

**Comment by ridvansalihkuzu** [comment]
Yes, you are right, data transform functions were applied to 4D arrays only for demonstration purposes on [TIME, CHANNELS, WIDTH, HEIGHT]. But it seems that the flip functions perform in the wrong axes, thank you for reporting this issue. We will make the corections accordingly.  

`data_transform.py` functions in the notebook are aimed to give very basic insights about data augmentation, and only `random_crop`, `rotation`, `padding` and `normalization` have been demonstrated without any optimization. Please feel free to implement your own data transformation/augmentation functions.

**Comment by crlna16** [comment]
Yes thank you, we came across while trying to build our custom data transform and just wanted to let you know :)
